internal/commands/resolve.go

```diff
@@ -246,15 +267,16 @@ func getCalendarEvent(
 	if err != nil {
 		logger.Error(
 			ctx,
-			"command/resolve.getCalendarEvent calendar.create_calendar_event error",
+			log.Trace(),
+			log.Reason("getCalendarEvent CreateCalendarEvent"),
 			log.NewValue("error", err),
 		)
 		return nil, err
 	}
 	return calendarEvent, err
 }
 
-func createResolveChannelAttachment(inc model.Incident, userName string, event *model.Event) slack.Attachment {
+func createResolveChannelAttachment(inc model.Incident, incidentID int64, userName string, event *model.Event) slack.Attachment {
```

Comment:
Same as the other files, you don't need the parameter incidentID, you can use `inc.Id` if you send the return of `repository.GetIncident`